src/json_value_module.F90

```diff
@@ -6965,7 +6964,7 @@ subroutine get_chars_from_array(json, element, i, count)
         if (.not. initialized) then
             ! string length long enough to hold the longest one
             allocate( character(kind=CK,len=max_len) :: vec(count) )
```

Comment:
This is probably causing test to fail on Travis (with the older gfortran). See #265.